testing/mesa/APKBUILD

```diff
@@ -0,0 +1,233 @@
+# Maintainer: Jens Staal <staal1978@gmail.com>
+# Adopted from: Natanael Copa <ncopa@alpinelinux.org>
+pkgname=mesa
+pkgver=13.0.1
+pkgrel=0
+pkgdesc="Mesa DRI OpenGL library"
+url="http://www.mesa3d.org"
+arch="all"
+license="LGPL"
+depends=
+
+subpackages="$pkgname-dev
+	$pkgname-dri-ati:ati
+	$pkgname-dri-nouveau:nouveau
+	$pkgname-dri-swrast:swrast
+	$pkgname-dri-vmwgfx:vmwgfx
+	$pkgname-dricore $pkgname-glapi $pkgname-egl $pkgname-gl $pkgname-gles
+	$pkgname-xatracker $pkgname-osmesa $pkgname-gbm
+	$pkgname-libwayland-egl:_wayland
+	"
+
+depends_dev="libdrm-dev dri2proto libx11-dev libxext-dev libxxf86vm-dev
+	libxdamage-dev libxfixes-dev libxcb-dev glproto dri3proto presentproto
+	libxshmfence-dev"
+makedepends="$depends_dev expat-dev xextproto python2 libxt-dev makedepend
+	talloc-dev py-libxml2 flex bison llvm-dev eudev-dev libvdpau-dev
+	libxvmc-dev gettext zlib-dev wayland-dev libelf-dev py-mako
+	autoconf automake libtool"
+source="https://mesa.freedesktop.org/archive/${pkgver}/mesa-$pkgver.tar.xz
+	glx_ro_text_segm.patch
+	musl-fixes.patch
+	"
+
+_dri_driverdir=/usr/lib/xorg/modules/dri
+_dri_drivers="r200,radeon,nouveau,swrast"
+_gallium_drivers="r300,r600,radeonsi,nouveau,svga,swrast,virgl"
+
+_builddir="$srcdir/mesa-$pkgver"
+
+_intel_dri=
+case "$CARCH" in
+x86*)
+	_dri_drivers="${_dri_drivers},i915,i965"
+	subpackages="$subpackages $pkgname-dri-intel:intel"
+	export _endian="LITTLEENDIAN_CPU"
```

Comment:
OK this was a "fossil" from when I tried to overide a failure of the build to recognize which endian there was. I can remove the export.